metadata.yaml

```diff
@@ -13,7 +13,7 @@ resources:
   oci-image:
     type: oci-image
     description: OCI image for hydra container
-    upstream-source: oryd/hydra:v2.0.2
+    upstream-source: oryd/hydra:v2.0.3
```

Comment:
In the future, please make the PR's simplers. Don't mix in new features with refactoring.